src/js/graphControl.js

```diff
@@ -1,163 +1,332 @@
-let series_type = 'line';
-let curentSeriesType;
-let traceType = ["line","scatter","bar","lineWithErrorBar","scatterWithErrorBar"];
-let dataPoints = [[5,7],[7,8],[15,10],[20,18],[17,21]]
-const graphControl = document.querySelector('#GraphControl');
-graphControl.dataZoom = [
-  {
-    type: 'inside'
-  }
-]
-graphControl.tooltip = {
-  axisPointer: {
-    type: "cross"
+let traceType = ['line', 'scatter', 'bar', 'lineWithErrorBar', 'scatterWithErrorBar'];
+/*---------------------sample graph for graph control start--------------------*/
+const dataPoints = [[5,7],[7,8],[15,10],[20,18],[17,21]]; 
+const getErrorData = () =>{
```

Comment:
@sagarbag6132401 please provide documentation

Reply:
Updated the documentation